### PR TITLE
Document change

### PR DESCRIPTION
**Pull request by preethi826**
change the formatting as an Initial PR. I will also create follow-up PR in the change in Linux instruction.

### CODE REVIEW

File: sdk/samples/iot/README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ To run the samples, ensure you have the following programs and tools installed o
 
 - Have the following build environment setup:
 
-    <details><summary><i>Instructions:</i></summary>
+    <summary><i>Instructions:</i></summary>
```
Comment (danewalton) [inline sdk/samples/iot/README.md:95]: please remove this line. You can also remove the `<p>` tags surrounding the text. 

[I have an example here for how to indent the text to look like the following:](https://github.com/danewalton/azure-sdk-for-c/blob/57efa7d640b6ead4d9200dbe1e374fee6f86952e/sdk/samples/iot/README.md?plain=1#L93-L106)

image

File: sdk/samples/iot/README.md [hunk 2]
```diff
@@ -107,12 +107,12 @@ To run the samples, ensure you have the following programs and tools installed o
...
 
     </p>
-    </details>
+    
 
 - Have [Git](https://git-scm.com/download) installed.
 - Have Microsoft [vcpkg](https://github.com/microsoft/vcpkg) package manager and [Eclipse Paho MQTT C client](https://www.eclipse.org/paho/) installed. This installation may take an extended amount of time (~15-20 minutes).
 
-    <details><summary><i>Instructions:</i></summary>
+    <summary><i>Instructions:</i></summary>
```
Comment (danewalton) [inline sdk/samples/iot/README.md:115]: remove and indent like the previous comment

File: sdk/samples/iot/README.md [hunk 3]
```diff
@@ -138,11 +138,11 @@ To run the samples, ensure you have the following programs and tools installed o
     ```
 
     </p>
-    </details>
+   
 
 - Have OpenSSL installed.
 
-    <details><summary><i>Instructions:</i></summary>
+    <summary><i>Instructions:</i></summary>
```
Comment (danewalton) [inline sdk/samples/iot/README.md:145]: remove like previous

File: sdk/samples/iot/README.md [hunk 4]
```diff
@@ -341,7 +341,7 @@ Set the following environment variables for all samples:
 
 Set the following environment variables if running any of these samples: `paho_iot_hub_c2d_sample`, `paho_iot_hub_methods_sample`, `paho_iot_hub_telemetry_sample`, `paho_iot_hub_twin_sample`, `paho_iot_pnp_sample`, `paho_iot_pnp_component_sample`
 
-<details><summary><i>Instructions to set environment variables for IoT Hub X.509 Certificate samples:</i></summary>
+<summary><i>Instructions to set environment variables for IoT Hub X.509 Certificate samples:</i></summary>
```
Comment (danewalton) [inline sdk/samples/iot/README.md:344]:
```suggestion
##### Set environment variables for IoT Hub X.509 Certificate samples
```

Comment (danewalton) [reply sdk/samples/iot/README.md:344]: and remove `<p>` tag below. All `<p>` tags can be removed in this document assuming proper indentation

File: sdk/samples/iot/README.md [hunk 5]
```diff
@@ -368,13 +368,13 @@ Set the following environment variables if running any of these samples: `paho_i
...
 </p>
-</details>
+
 
 #### IoT Provisioning X.509 Certificate Sample
 
 Set the following environment variables if running the sample:  `paho_iot_pnp_with_provisioning_sample`, `paho_iot_provisioning_sample`
 
-<details><summary><i>Instructions to set environment variables for DPS X.509 Certificate sample:</i></summary>
+<summary><i>Instructions to set environment variables for DPS X.509 Certificate sample:</i></summary>
```
Comment (danewalton) [inline sdk/samples/iot/README.md:377]:
```suggestion
##### Set environment variables for DPS X.509 Certificate sample
```

File: sdk/samples/iot/README.md [hunk 6]
```diff
@@ -401,13 +401,13 @@ Set the following environment variables if running the sample:  `paho_iot_pnp_wi
...
 </p>
-</details>
+
 
 #### IoT Hub Symetric Key (SAS) Sample
 
 Set the following environment variables if running the sample:  `paho_iot_hub_sas_telemetry_sample`
 
-<details><summary><i>Instructions to set environment variables for IoT Hub Symmetric Key sample:</i></summary>
+<summary><i>Instructions to set environment variables for IoT Hub Symmetric Key sample:</i></summary>
```
Comment (danewalton) [inline sdk/samples/iot/README.md:410]:
```suggestion
##### Set environment variables for IoT Hub Symmetric Key sample
```

File: sdk/samples/iot/README.md [hunk 7]
```diff
@@ -437,13 +437,13 @@ Set the following environment variables if running the sample:  `paho_iot_hub_sa
...
 </p>
-</details>
+
 
 #### IoT Provisioning Symmetric Key (SAS) Sample
 
 Set the following environment variables if running the sample: `paho_iot_provisioning_sas_sample`
 
-<details><summary><i>Instructions to set environment variables for DPS Symmetric Key sample:</i></summary>
+<summary><i>Instructions to set environment variables for DPS Symmetric Key sample:</i></summary>
```
Comment (danewalton) [inline sdk/samples/iot/README.md:446]:
```suggestion
##### Set environment variables for DPS Symmetric Key sample
```

File: sdk/samples/iot/README.md [hunk 8]
```diff
@@ -517,21 +517,21 @@ This section provides an overview of the different samples available to run and
 
   This [sample](https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/iot/paho_iot_hub_c2d_sample.c) receives incoming cloud-to-device (C2D) messages sent from the Azure IoT Hub to the device. It will successfully receive up to 5 messages sent from the service. If a timeout occurs while waiting for a message, the sample will exit. X509 authentication is used.
 
-  <details><summary><i>How to interact with the C2D sample:</i></summary>
+  <summary><i>How to interact with the C2D sample:</i></summary>
```
Comment (danewalton) [inline sdk/samples/iot/README.md:520]:
```suggestion
  #### How to interact with the C2D sample
```

File: sdk/samples/iot/README.md [hunk 9]
```diff
@@ -517,21 +517,21 @@ This section provides an overview of the different samples available to run and
...
+
 
 ### IoT Hub Methods Sample
 
 - *Executable:* `paho_iot_hub_methods_sample`
 
   This [sample](https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/iot/paho_iot_hub_methods_sample.c) receives incoming method commands invoked from the the Azure IoT Hub to the device. It will successfully receive up to 5 method commands sent from the service. If a timeout occurs while waiting for a message, the sample will exit. X509 authentication is used.
 
-  <details><summary><i>How to interact with the Methods sample:</i></summary>
+  <summary><i>How to interact with the Methods sample:</i></summary>
```
Comment (danewalton) [inline sdk/samples/iot/README.md:534]:
```suggestion
  #### How to interact with the Methods sample
```

File: sdk/samples/iot/README.md [hunk 10]
```diff
@@ -565,7 +565,7 @@ This section provides an overview of the different samples available to run and
 
   This [sample](https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/iot/paho_iot_hub_twin_sample.c) utilizes the Azure IoT Hub to get the device twin document, send a reported property message, and receive up to 5 desired property messages. If a timeout occurs while waiting for a message from the Azure IoT Hub, the sample will exit. Upon receiving a desired property message, the sample will update the twin property locally and send a reported property message back to the service. X509 authentication is used.
 
-  <details><summary><i>How to interact with the Twin sample:</i></summary>
+ <summary><i>How to interact with the Twin sample:</i></summary>
```
Comment (danewalton) [inline sdk/samples/iot/README.md:568]:
```suggestion
 #### How to interact with the Twin sample
```

File: sdk/samples/iot/README.md [hunk 11]
```diff
@@ -583,13 +583,13 @@ This section provides an overview of the different samples available to run and
...
   </p>
-  </details>
+
 
 ### IoT Plug and Play Sample
 
   This [sample](https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/iot/paho_iot_pnp_sample.c) connects an IoT Plug and Play enabled device (a thermostat) with the Digital Twin Model ID (DTMI) detailed [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json). If a timeout occurs while waiting for a message from the Azure IoT Explorer, the sample will continue. If 3 timeouts occur consecutively, the sample will disconnect. X509 authentication is used.
 
-  <details><summary><i>How to interact with the IoT Plug and Play sample:</i></summary>
+ <summary><i>How to interact with the IoT Plug and Play sample:</i></summary>
```
Comment (danewalton) [inline sdk/samples/iot/README.md:592]:
```suggestion
 #### How to interact with the IoT Plug and Play sample
```

File: sdk/samples/iot/README.md [hunk 12]
```diff
@@ -624,7 +624,7 @@ This section provides an overview of the different samples available to run and
   - [Temperature Sensor 2](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json)
   - The properties, commands, and telemetry of the Temperature Controller itself.
 
-  <details><summary><i>How to interact with the IoT Plug and Play Multiple Component sample:</i></summary>
+ <summary><i>How to interact with the IoT Plug and Play Multiple Component sample:</i></summary>
```
Comment (danewalton) [inline sdk/samples/iot/README.md:627]:
```suggestion
 #### How to interact with the IoT Plug and Play Multiple Component sample
```

**Review by danewalton — CHANGES_REQUESTED** [review]

File: sdk/samples/iot/README.md (hunk 1 above)
Comment (preethi826) [reply sdk/samples/iot/README.md:95]: So it looks like we have to remove some instruction lines.

File: sdk/samples/iot/README.md (hunk 4 above)
Comment (preethi826) [reply sdk/samples/iot/README.md:344]: I see multiple places in this document and also before the instruction. do I need to remove whatever is there before instruction?

File: sdk/samples/iot/README.md (hunk 1 above)
Comment (preethi826) [reply sdk/samples/iot/README.md]: Modified

File: sdk/samples/iot/README.md (hunk 4 above)
Comment (preethi826) [reply sdk/samples/iot/README.md:326]: Modified

File: sdk/samples/iot/README.md (hunk 5 above)
Comment (preethi826) [reply sdk/samples/iot/README.md:356]: Modified

**Pull request closed without merging**